pom.xml

```diff
@@ -140,6 +140,12 @@ under the License.
       <artifactId>maven-scm-api</artifactId>
       <version>${scmVersion}</version>
     </dependency>
+    <dependency>
+      <groupId>javax.inject</groupId>
+      <artifactId>javax.inject</artifactId>
+      <version>1</version>
+      <scope>Provided</scope>
```

Comment:
Provided -> provided

Reply:
Oops. surprised that didn't break the build. Do we really allow any string for scope?